cmd/up/activate.go

```diff
@@ -103,8 +103,11 @@ func (up *upContext) activate() error {
 		}
 	}
 
-	go up.initializeSyncthing()
-
+	go func() {
+		if err := up.initializeSyncthing(); err != nil {
+			oktetoLog.Infof("could not initialize syncthing: %s", err)
```

Comment:
@jLopezbarb should we add some mechanism to stop the command if this fails? 

@abs007 don't mind about this, it's out of scope for this PR

Reply:
It's just launching the syncthing process, that we have previously downloaded to make it compatible with the user OS, so this shouldn't fail in any circunstances.

Otherwise, there is a mechanism to wait for 1min and if syncthing doesn't start we show a proper error to the user here: https://github.com/okteto/okteto/pull/3278/files/c0736885136289a5a7130639c9bb63633abdac0c#diff-c39b965c9f4376229cb1d945d1dcddaca021d99664fe2c5674aee1c278999600R155